iree/turbine/aot/exporter.py

```diff
@@ -49,6 +50,21 @@
 SaveableTarget = Union[str, Path, None, Output]
 
 
+class DeviceAffinity:
+    """This is used to provide device affinities to exported function arguments."""
+
+    def __init__(self, moniker: str):
```

Comment:
Make the moniker an int. All cases around just specify it that way anyway.

Reply:
I changed it to `ordinal: int`.